storm-server/src/test/java/org/apache/storm/utils/ServerUtilsTest.java

```diff
@@ -162,6 +185,191 @@ public void testGetUserId() throws Exception {
         int uid1 = ServerUtils.getUserId(null);
         Path p = Files.createTempFile("testGetUser", ".txt");
         int uid2 = ServerUtils.getPathOwnerUid(p.toString());
+        if (!p.toFile().delete()) {
+            LOG.warn("Could not delete tempoary file {}", p);
+        }
         assertEquals("User UID " + uid1 + " is not same as file " + p.toString() + " owner UID of " + uid2, uid1, uid2);
     }
+
+    @Test
+    public void testIsAnyProcessPosixProcessPidDirAlive() throws IOException {
```

Comment:
Isn't this already present? 
https://github.com/apache/storm/blob/master/storm-server/src/test/java/org/apache/storm/utils/ServerUtilsTest.java#L184

Reply:
Yes. Something is not right. I will do a force push with all changes.

Reply:
You need to rebase on latest master. Then you should be able to see the conflicts/duplicates